README.md

```diff
@@ -119,7 +134,7 @@ require("nvim-paredit").setup({
 })
 ```
 
-Or by calling the `add_language_extension` API directly. This would be the recommended approach for extension plugin authors.
+Or by calling the `add_language_extension` API directly before the setup. This would be the recommended approach for extension plugin authors.
```

Comment:
I think it would be tricky to enforce this is called before the setup. We should probably change the `add_language_extension` API to perform the necessary setup work. Can be done in a future PR.

Reply:
yes, it became tricky to support filetypes option and an extension api together